docs/src/components/HomeBanner.astro

```diff
@@ -0,0 +1,20 @@
+---
+import CallToAction from '@astrojs/starlight/components/CallToAction.astro'
+---
+
+Style once, use everywhere
+<h1>Hello world</h1>
+
+<CallToAction variant='primary' link='/style'>Testing</CallToAction>
```

Comment:
Just leaving a comment so we dont forget to remove this testing thing

I'm also not sure if I like the CTA gradient but that's personal taste so not a blocker.